test/e2e/network/util_iperf.go

```diff
@@ -23,8 +23,6 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
-
-	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```

Comment:
"k8s.io/kubernetes/test/e2e/framework" is needed to use framework.Logf and framework.Failf.

Reply:
Thanks for your suggestion.I changed it.